contract/contracts/PixelAvatars.sol

```diff
@@ -29,6 +31,8 @@ contract PixelAvatars is
         __ERC721_init("Pixel Avatars", "PXLDEV");
         __Ownable_init();
 
+        setRoyalties(10000); // On initialize set 10% royalty fee
```

Comment:
@carlomigueldy this needs to be set to 1000 for a 10% royalty fee.

Reply:
Hey, thanks again for the spot :) @itstargetconfirmed